twilio-api/twilio-filters.php

```diff
@@ -182,20 +182,12 @@ function send_notification_on_channels( $user_id, $notification, $notification_t
     if ( channel_notification_enabled() && isset( $notification['post_id'] ) ) {
         $post_id = $notification['post_id'];
 
-        // Construct required user object structure.
-        $user = (object) [
-            'dt_id' => $user_id,
-            'sys_type' => 'wp_user',
-            'type' => 'member'
-        ];
-
-        // Attempt to source any associated phone numbers for given user.
-        $phone_numbers = Disciple_Tools_Twilio_API::get_user_phone_numbers( $user );
-        if ( !empty( $phone_numbers ) ) {
-            $phone_number = $phone_numbers[0];
+        // Ensure a valid source user id has been specified.
+        if ( !empty( $notification['source_user_id'] ) ) {
```

Comment:
@kodinkat this gets the source user, the one triggering the notifications.
But this isn't who we are wanting to send to notification to.

Reply:
fixed in https://github.com/DiscipleTools/disciple-tools-channels-twilio/pull/23/commits/796569eb9a11ac1f4e6225e0c2c76a3853949a8f